jsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+    "compilerOptions": {
+        "target": "es2017",
+        "allowSyntheticDefaultImports": false,
+        "baseUrl": "./",
```

Comment:
please make the changes in the jest.config.js 
https://github.com/wednesday-solutions/nodejs-hapi-template/blob/master/jest.config.js